include/fmt/core.h

```diff
@@ -2002,11 +2008,11 @@ using format_args = basic_format_args<format_context>;
 // We cannot use enum classes as bit fields because of a gcc bug
 // https://gcc.gnu.org/bugzilla/show_bug.cgi?id=61414.
 namespace align {
-enum type { none, left, right, center, numeric };
+enum type : unsigned { none, left, right, center, numeric };
```

Comment:
Why unsigned?

Reply:
As per commit message: to fix the -Wsigned-enum-bitfield warnings.